packages/cozy-harvest-lib/src/components/AccountCreator.jsx

```diff
@@ -36,10 +36,8 @@ export class AccountCreator extends PureComponent {
     const { account, konnector, submitting } = this.props
     return (
       <AccountForm
-        fields={konnector.fields}
+        konnector={konnector}
```

Comment:
🎉 

I think it is much simpler mentally to pass around known objects and it avoids creating a lot of props.

Reply:
Totally agree, it was a matter of time :)

Reply:
Funny seeing this after the other review :p It's ok if the object is *known*, but for me at least `konnector` is opaque and I have no idea what is or isn't in it, so i actually prefer being explicit about the props. It can also help avoid extra renders when used in PureComponents.